packages/form-js-editor/src/render/components/properties-panel/entries/InputKeyValuesSourceEntry.js

```diff
@@ -10,12 +10,18 @@ export default function InputKeyValuesSourceEntry(props) {
     id
   } = props;
 
+  const schema = '[{"label": "dollar", "value": "$"}]';
+
+  const description = <div>
+    Define which input property to populate the values from. The property must follow this schema: <code>{schema}</code>.
```

Comment:
Should we maybe add the description as mentioned in the issue (https://github.com/bpmn-io/form-js/issues/305)? So having "The property must follow ..." as another paragraph and add some indentation to the code example. 

![image](https://user-images.githubusercontent.com/9433996/185421918-70a2e583-a24c-4dc1-b1c5-7cba1360bbe8.png)

Reply:
I like this one much more 👍 

Ideally, this should be done by the core (description) styles, yeah, to align it across our tools.

Reply:
We could also split this PR into two parts (bug fix, description updates) and, make progress with the bug fix faster + deliver the style + description updates in another PR.